__tests__/app.test.js

```diff
@@ -37,3 +38,39 @@ describe("/api/topics", () => {
     });
   });
 });
+describe("/api/articles/:atricle_id", () => {
```

Comment:
Be careful with typos - I know this is only a describe label for a test, but try to correct it soon please